lib/client/chart.js

```diff
@@ -14,24 +14,39 @@ function init (client, d3, $) {
   var utils = client.utils;
   var renderer = client.renderer;
 
+  var defs = d3.select('body').append('svg').append('defs');
+
   // add defs for combo boluses
   var dashWidth = 5;
-  d3.select('body').append('svg')
-    .append('defs')
-    .append('pattern')
-      .attr('id', 'hash')
-      .attr('patternUnits', 'userSpaceOnUse')
-      .attr('width', 6)
-      .attr('height', 6)
-      .attr('x', 0)
-      .attr('y', 0)
-      .append('g')
-        .style('fill', 'none')
-        .style('stroke', '#0099ff')
-        .style('stroke-width', 2)
-      .append('path').attr('d', 'M0,0 l' + dashWidth + ',' + dashWidth)
-      .append('path').attr('d', 'M' + dashWidth + ',0 l-' + dashWidth + ',' + dashWidth);
+  defs.append('pattern')
+    .attr('id', 'hash')
+    .attr('patternUnits', 'userSpaceOnUse')
+    .attr('width', 6)
+    .attr('height', 6)
+    .attr('x', 0)
+    .attr('y', 0)
+    .append('g')
+      .style('fill', 'none')
+      .style('stroke', '#0099ff')
+      .style('stroke-width', 2)
+    .append('path').attr('d', 'M0,0 l' + dashWidth + ',' + dashWidth)
+    .append('path').attr('d', 'M' + dashWidth + ',0 l-' + dashWidth + ',' + dashWidth);
  
+  // arrow head
+  defs.append("marker")
+    .attr({
+      "id":"arrow",
+      "viewBox":"0 -5 10 10",
+      "refX":5,
+      "refY":0,
+      "markerWidth":8,
+      "markerHeight":8,
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1562495607/issues/source?bid=2452379&fileBranchId=3019946#l43)